static_inverse_kinematics.py

```diff
@@ -63,21 +66,46 @@ def __init__(
         self.c3d_path_file = c3d_path_file
 
         self.biorbd_model = biorbd.Model(self.biorbd_model_path)
-        self.c3d_data = C3dData(self.c3d_path_file, self.biorbd_model)
+        self.c3d = c3d(self.c3d_path_file)
 
-        self.list_model_markers = [
+        self.marker_names = [
             self.biorbd_model.markerNames()[i].to_string() for i in range(len(self.biorbd_model.markerNames()))
         ]
-        self.xp_markers = self.c3d_data.trajectories
+        self.xp_markers = self.get_marker_trajectories()
         self.nb_q = self.biorbd_model.nbQ()
-        self.nb_frames = self.c3d_data.nb_frames
+        self.nb_frames = self.c3d["parameters"]["POINT"]["FRAMES"]["value"][0]
         self.nb_markers = self.biorbd_model.nbMarkers()
 
+        self._get_idx_to_remove()
```

Comment:
idx_to_remove is missing in the docstring

Reply:
done